src/components/Menu/MenuItem.tsx

```diff
@@ -143,14 +147,20 @@ class MenuItem extends UIComponent<Extendable<IMenuItemProps>, any> {
     const { children, content, icon } = this.props
 
     return (
-      <ElementType className={classes.root} {...accessibility.attributes.root} {...rest}>
+      <ElementType
+        className={classes.root}
+        {...accessibility.attributes.root}
+        {...accessibility.handlers.root}
+        {...rest}
+      >
         {childrenExist(children) ? (
           children
         ) : (
           <a
             className={cx('ui-menu__item__anchor', classes.anchor)}
             onClick={this.handleClick}
             {...accessibility.attributes.anchor}
+            {...accessibility.handlers.anchor}
```

Comment:
`handlers` seems to be a name that quite loosely describes the purpose of this member - so it is hard to get the intent of the client code by just reading it. Maybe we could make it more expressive by introducing name like `keyHandlers`, which it, in fact, is

Reply:
agree, I'll rename it if everyone is ok with that.